Examples/ScalarFieldCosmo/ScalarFieldCosmoLevel.cpp

```diff
@@ -305,9 +306,24 @@ void CosmoLevel::specificPostTimeStep()
             std::array<double, CH_SPACEDIM> extraction_points = {
                 0., m_p.L / 2, m_p.L / 2}; // specified point {x \in [0,L],y \in
                                            // [0,L], z \in [0,L]}
-            CustomExtraction extraction(c_rho, m_p.lineout_num_points, m_p.L,
-                                        extraction_points, m_dt, m_time);
-            extraction.execute_query(&interpolator, m_p.data_path + "lineout");
+            // rho lineout
+            CustomExtraction rho_extraction(c_rho, m_p.lineout_num_points,
+                                            m_p.L, extraction_points, m_dt,
+                                            m_time);
+            rho_extraction.execute_query(&interpolator,
+                                         m_p.data_path + "rho_" + "lineout");
+            // Ham lineout
```

Comment:
Will this run on a restart? Maybe it does, but I think it might be better to put it into the restart function, as I did on the ScalarFieldBH example - take a look at that and see what you think. I don't think we need this at every tilmestep.